soh/soh/Enhancements/randomizer/3drando/hint_list.cpp

```diff
@@ -2493,90 +2493,94 @@ void StaticData::HintTable_Init() {
+                                                 /*german*/ "Man erzählt sich, daß #Bongo Bongo# #[[1]]# hielte.",
                                                  /*french*/ "Selon moi, #Bongo Bongo# possède #[[1]]#.", {QM_RED, QM_GREEN}),
                                               // /*spanish*/#Bongo Bongo# porta #[[1]]#.
                                               {},
                                               {CustomMessage("They say that the #Phantom Shadow Beast# holds #[[1]]#.",
-                                                  /*german*/ "Man erzählt sich, daß das #Phantomschattenbiest# hält #[[1]]# lehre.",
+                                                  /*german*/ "Man erzählt sich, daß das #Phantomschattenbiest# #[[1]]# hielte.",
                                                   /*french*/ "Selon moi, le #monstre de l'ombre# possède #[[1]]#.", {QM_RED, QM_GREEN})});
                                                // /*spanish*/la #alimaña oscura espectral# porta #[[1]]#.
 
     hintTextTable[RHT_TWINROVA] = HintText(CustomMessage("They say that #Twinrova# holds #[[1]]#.",
-                                              /*german*/ "Man erzählt sich, daß #Twinrova# hält #[[1]]# lehre.",
+                                              /*german*/ "Man erzählt sich, daß #Twinrova# #[[1]]# hielte.",
                                               /*french*/ "Selon moi, #Twinrova# possède #[[1]]#.", {QM_RED, QM_GREEN}),
                                            // /*spanish*/#Birova# porta #[[1]]#.
                                            {},
                                            {CustomMessage("They say that the #Sorceress Sisters# hold #[[1]]#.",
-                                               /*german*/ "Man erzählt sich, daß die #Hexenschwestern# halten #[[1]]# lehre.",
+                                               /*german*/ "Man erzählt sich, daß die #Hexenschwestern# #[[1]]# hielten.",
                                                /*french*/ "Selon moi, #les sorcières jumelles# possède #[[1]]#.", {QM_RED, QM_GREEN})});
                                             // /*spanish*/las #hermanas hechiceras# portan #[[1]]#.
 
-    hintTextTable[RHT_GIFT_FROM_RAURU] = HintText(CustomMessage("They say that the #Sage of Light# gifts @ #[[1]]#.", {QM_RED, QM_GREEN}),
-                                           {},
-                                           {CustomMessage("They say that #a former owl# gifts @ #[[1]]#.", {QM_RED, QM_GREEN})});
+    hintTextTable[RHT_GIFT_FROM_RAURU] = HintText(CustomMessage("They say that the #Sage of Light# gifts @ #[[1]]#.",
+                                                     /*german*/ "Man erzählt sich, daß die #Weise des Lichts# #[[1]]# schenke.",
+                                                     /*french*/ "Selon moi, #la Sage de la Lumière# donne #[[1]]#.", {QM_RED, QM_GREEN}),
```

Comment:
Heya, you made a typo for the french, "le Sage" and not "la Sage"

Reply:
> Heya, you made a typo for the french, "le Sage" and not "la Sage"

I forgot that Rauru is actually a male character.